src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -54,17 +69,31 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   const containerCssVars = useCss({
     "--event--margin-left": `${eventMarginLeftPx}px`,
     "--event--width": `${eventWidthPx}px`,
+    "--event--expanded-width": `${expandedEventPx}px`,
   });
 
+  const isEventLong =
+    event.duration_minutes > SCHEDULE_MIN_LONG_EVENT_LENGTH_MIN;
+  const isEventShort =
+    event.duration_minutes <= SCHEDULE_SHORT_EVENT_LENGTH_MIN;
```

Comment:
Do we have it covered if an event is normal size? 15 - 45 mins long?

Reply:
Yeah, everything in between will display collapsed & expand on hover 
![image](https://user-images.githubusercontent.com/56254806/125574442-ffa6d4e2-4f3e-4cee-aa06-7cfe806ef186.png)

Reply:
minor fix 1540077